<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_icon1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 표시 위치</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_run_email">
       <property name="text">
        <string>Run E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run_musicplayer">
       <property name="text">
        <string>Run MusicPlayer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run_news">
       <property name="text">
        <string>Run News</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_icon3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 표시 위치</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지 표시 위치</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
